util/tune/current_settings.rb

```diff
@@ -40,29 +43,29 @@
 #   value = puppetserver_jruby_max_active_instances
 #
 def puppetserver_jruby_max_active_instances
-  if File.exist? PE_PUPPET_SERVER_CONF
-    line = File.open(PE_PUPPET_SERVER_CONF).grep(/max-active-instances/)[0]
-    value = line.scan(/\d+/).first unless line.nil?
-  end
-
-  unless value
-    # TODO: eliminate bash
-    command = "facter processorcount"
-    output = `#{command}`
-
-    if output
-      # See the docs link in the description above
-      # The default used in PE is the number of CPUs - 1, expressed as $::processorcount - 1.
-      # One instance is the minimum value and four instances is the maximum value.
-      #
-      # TODO: Determine if / where the default value for the 'jruby_max_active_instances' is set
-      #   See https://tickets.puppetlabs.com/browse/SLV-530
-      num_cores = output.to_i
-      value = [[(num_cores - 1), MIN_DEFAULT_JRUBIES].max, MAX_DEFAULT_JRUBIES].min.to_s
-    else
-      value = NA
-    end
-
+  return NA unless File.exist? PE_PUPPET_SERVER_CONF
+
+  conf = File.read(PE_PUPPET_SERVER_CONF)
+  match_pattern = /^\s*max-active-instances: (\d+)/
+  conf_scan = conf.scan(match_pattern)
+  value = conf_scan.last ? conf_scan.last[0] : nil
```

Comment:
Does John's request for line 225 (https://github.com/puppetlabs/gatling-puppet-load-test/pull/357#discussion_r348170034) apply here as well?